packages/components/src/presentation-components/instance-filter-builder/PresentationFilterBuilder.tsx

```diff
@@ -117,6 +117,8 @@ export interface PresentationFilterBuilderValueRendererProps extends PropertyFil
   descriptor: Descriptor;
   /** Keys used to create the descriptor. */
   descriptorInputKeys?: Keys;
+  /** Currently selected classes. */
```

Comment:
I think it would be great to provide more docs here. E.g.:
- How this prop affects the component's behavior?
- What's the default when prop is not provided?

Basically, try to think as an API consumer and make sure the docs are sufficient for you to understand the meaning of this prop.

Reply:
Changed